README.md

```diff
@@ -1,83 +1,186 @@
+[![Build Status](https://github.com/boldlink/terraform-aws-ssm/actions/workflows/checkov.yaml/badge.svg)](https://github.com/boldlink/terraform-aws-ssm/actions)
+[![Build Status](https://github.com/boldlink/terraform-aws-ssm/actions/workflows/auto-merge.yaml/badge.svg)](https://github.com/boldlink/terraform-aws-ssm/actions)
+[![Build Status](https://github.com/boldlink/terraform-aws-ssm/actions/workflows/auto-badge.yaml/badge.svg)](https://github.com/boldlink/terraform-aws-ssm/actions)
 
 [<img src="https://avatars.githubusercontent.com/u/25388280?s=200&v=4" width="96"/>](https://boldlink.io)
 
-# Terraform  module \<PROVIDER>-\<MODULE> Terraform module
-
-## How to use this template -- DELETE THIS SECTION BEFORE PR
-1. Create your new module repository by using terraform only (see SOP) and make sure to use this template.
-2. Add your Terraform code in any branch other than `main/master`
-3. Change the `<REPO_NAME>` value for any badges in the `README.md` files in the root `examples` and `modules` folders.
-4. Add nested modules in the `modules` folder, or `DELETE` the nested folder if not used.
-    * _Note: you will also maintain the nested modules full `README.md` files, remember nested modules can be used on their own._
-5. Add examples in the `examples` folder.
-    * _Note: you can have as many examples as you want, but two are required._
-        * _minimum - this is the example with the minimum code to use the module._
-        * _complete - this is the example with all features for a single module used (the most common use)._
-6. Run `pre-commit run --all-files` to update the `README` and fix any issues.
-    * _Note: make sure your IDE tool uses spaces and not tabs specially on `yaml` files._
-7. Run `checkov` or `terrascan` tool and make sure to add the log to the PR (something to automate).
-    * _Note: make sure to scan your module nested modules and examples (great candidate for a makefile action/script and automation)._
-8. Open a pull request into the default branch (usually `main`) and have it reviewed. don't forget to add the security scan logs.
-    * _Note: make sure to add the nested modules README's to the pre-commit config so they are also updated and validated._
-9. If you have been assigned a reviewer DM the reviewer, or the channel if it has been more than one day.
-10. Post to the channel news of the releases to the teams.
-
-## Description
-
-lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem.
-
-Examples available [`here`]github.com/boldlink/<REPO_NAME>//tree/main/examples)
+# Terraform AWS Systems Manager module
+This module simplifies the process of creating AWS Systems Manager sessions, offering options to enable session encryption, CloudWatch Logs encryption and S3 Logs encryption. The module also assists in generating run command documents for both Linux and Windows instances. This assists one to execute commands simultaneously across instances managed by AWS Systems Manager.
```

Comment:
Also mention this
```
**NOTE:** 
* You can change the default AWS SSM session `SSM-SessionManagerRunShell` by importing it first to your state and then apply the module changes.
* We highly recommend you to condition which session document for SSM Session you allow users to use, this will restrict users to start the SSM Sessions using their own custom documents or the default document (SSM-SessionManagerRunShell) otherwise you will make whatever session document you configured redundant, see the AWS documentation for this [here](https://docs.aws.amazon.com/systems-manager/latest/userguide/getting-started-sessiondocumentaccesscheck.html).
```